website/content/gateway/examples/series_of_transactions/repeatedly_reserve_money/repeatedly_reserve_money.md

```diff
@@ -45,8 +45,8 @@ curl -X POST \
   -d "card[expire_year]=2026"     \
   -H "Signature: <signing-api-key> RS256-hex <signature>"
 ```
-A first-in-series authorization can also be made using the {{% highlight_text %}}applepay{{% /highlight_text %}}, {{% highlight_text %}}googlepay{{% /highlight_text %}} or {{% highlight_text %}}mobilepayonline{{% /highlight_text %}} payment methods.
+A first-in-series authorization or debit can also be made using the {{% highlight_text %}}applepay{{% /highlight_text %}}, {{% highlight_text %}}googlepay{{% /highlight_text %}}, {{% highlight_text %}}mobilepayonline{{% /highlight_text %}}, {{% highlight_text %}}token{{% /highlight_text %}} or {{% highlight_text %}}vipps{{% /highlight_text %}} payment methods.
```

Comment:
Unfortunately, I cannot answer your excellent questions. For now I can only play the card that we already mentioned `mobilepayonline` here before.

Reply:
cc: @ab-clearhaus

Reply:
@AlexanderIversen I have to disappoint you this time around. At a recent planning meeting we decided to move on with this PR, even though we are not sure about series/subscriptions with MobilePay Online and Vipps. Even though the two payment methods might not offer series/subscriptions, it is technically possible to use our API to process series/subscriptions using credentials from the payment methods. Also, our API documentation already mentions `mobilepayonline` here before this PR, so the PR is only adding `vipps` to the list, i.e. the PR is not *significantly* adding more misinformation than -- perhaps -- already present.

Reply:
Okay that a decision made by the team. 👍 

Allow to ask a couple of questions after all. 
 * Does anyone monitor the approval rate on the potential Mobilepay series transactions? And what do we recommend our partners to do regarding the subsequent in series? 
* When MPO PAN then `card[]` and when MPO TAN then `token[]` or how have we implemented support for it? 
* And same questions as previously – do we allow it for both Recurring and UCOF? 
* Should the partner guideline be updated in relation to this? 

cc: @ct-clearhaus

Reply:
> * Does anyone monitor the approval rate on the potential Mobilepay series transactions? And what do we recommend our partners to do regarding the subsequent in series?

Nope. IIRC, in the old days, merchants were prohibited to do series processing on MPO (via a direct contract between MP and the merchant), but we allowed it anyway. Reason being that some merchants may have had a different contract, and we shouldn't adjust our API's abilities according to such contracts.
Likewise I'd say, the MPO tokens may allow for series processing — or they may not. I'd like that our API's abilities doesn't depend on what MPO does. If MPO tokens never allow for series processing and for sure never will (cf. scheme documentation/rules/...), I'm fine with removing the functionality — otherwise, I'd like the payment methods to be as equal as possible.

I don't think we have any recommendations in this regard.

> * When MPO PAN then `card[]` and when MPO TAN then `token[]` or how have we implemented support for it?

When MPO, use `mobilepayonline[]` — no matter if it's tokenized or not. For a subsequent-in-series, I assume that the payment method isn't MPO[^1], so it'll be either `token[]` or `card[]` — and hopefully it's a TAN so `token[]` would be appropriate.

[^1]: Because you don't send the cardholder to some MPO URL and get some MPO data blob back. Instead, as a TP, you stored the CAN (PAN or TAN) on file and then use it for the subsequent-in-series.

> * And same questions as previously – do we allow it for both Recurring and UCOF?

Yes, I don't see why we shouldn't. (Same reasoning as the first point.)

Reply:
I will merge the PR. We can still continue the discussion. Potential changes resulting from the discussion will get a new PR eventually.